### PR TITLE
Matching score of 0 should not exclude candidates

### PR DESCRIPTION
**Pull request by mattcherryfish**

### CODE REVIEW

File: packages/py-matching/py-matching.iml [hunk 1]
```diff
@@ -2,14 +2,16 @@
...
   <component name="NewModuleRootManager" inherit-compiler-output="true">
     <exclude-output />
-    <content url="file://$MODULE_DIR$" />
+    <content url="file://$MODULE_DIR$">
+      <sourceFolder url="file://$MODULE_DIR$/search/search" isTestSource="false" />
+    </content>
     <orderEntry type="inheritedJdk" />
     <orderEntry type="sourceFolder" forTests="false" />
-    <orderEntry type="library" name="Pipenv (py-cherrybrain) interpreter library" level="application" />
+    <orderEntry type="library" name="Python 3.7 interpreter library" level="application" />
```
Comment (EnriqueCherryfish) [inline packages/py-matching/py-matching.iml]: What are this changes?

File: packages/py-matching/search/search/conditions/dimensionsearch.py [hunk 2]
```diff
@@ -5,7 +5,8 @@
 class DimensionSearch:
     def __init__(self, parameters):
         self.parameters = parameters['dimensions']
-        self.number_of_target_levels = self.count_number_of_target_levels_in_dimension_search()
+        self.number_of_target_levels = self.count_number_of_target_levels_in_dimension_search(
+        )
```
Comment (EnriqueCherryfish) [inline packages/py-matching/search/search/conditions/dimensionsearch.py]: That space xD

File: packages/py-matching/py-matching.iml (hunk 1 above)
Comment (mattcherryfish) [reply packages/py-matching/py-matching.iml]: They're to do with my IDE, ignore them

File: packages/py-matching/search/search/conditions/dimensionsearch.py (hunk 2 above)
Comment (mattcherryfish) [reply packages/py-matching/search/search/conditions/dimensionsearch.py]: 😅

**Pull request merged**